### DISCUSSION TITLE
App won't recognize speed sensor on my Powercolor Hellhound 7900 XT GPU

### DISCUSSION THREAD

**Discussion by MEDAFAP** [Rem0o/FanControl.Releases#2415] [Q&A]
Fan Control can't seem to recognize a speed sensor on my GPU, but FANPWMIN0 does show up on HW Monitor.... any suggestions as to how to get Fan Control to see my GPU fans?  Thanks in advance.
![FanControl Screenshot](https://github.com/Rem0o/FanControl.Releases/assets/165120047/0bb83963-04b5-4747-a748-06a92d89259f)
![HWMonitor screenshot](https://github.com/Rem0o/FanControl.Releases/assets/165120047/e76b1877-e4ac-49eb-bebe-d1042c1c73b2)

**Comment by ionGL** [comment]
Chose manual control for the gpu fan, set it 50-100%, see what happens.

**Comment by Rem0o** [comment]
7900 XT is not currently natively supported properly.

You can give https://github.com/Rem0o/FanControl.ADLX a try.

**Reply by Rem0o** [reply]
Plan is to eventually merge this plugin into the main app, but there is still work/tests to be done before that happens.

**Comment by MEDAFAP** [comment]
Thank you for the link!

________________________________
